docs/UserGuide.md

```diff
@@ -210,7 +210,8 @@ Format: `addClaim INDEX iid/INSURANCE_ID cid/CLAIM_ID ca/CLAIM_AMOUNT_IN_DOLLARS
 3. `CLAIM_ID` : a unique id for each claim generated by the company and should be created externally by another
 organisation before the claim is added to this app.
 4. `CLAIM_AMOUNT_IN_DOLLARS_AND_CENTS` is the claim amount written in `DD.CC` format eg. `151.10`. So, it can contain
-any number of digits for dollars but must be exactly 2 digits for cents. The claim amount can be up till 1 million.
+any number of digits for dollars (leading zeroes will be trimmed), but must be exactly 2 digits for cents. If the claim amount
+is a whole number eg. $500, enter it as `500.00`. The claim amount can be up till 1 million Singapore Dollars (SGD).
```

Comment:
remove the denomination here, it will lead to more bugs reported. just say in the UG/DG the target profile is Singaporean Insurance Agents working in singapore then if they try something we can just point them to that.

Reply:
Ok, sounds good to me. I'll change it